CODE_OF_CONDUCT_fr.md

```diff
@@ -0,0 +1,52 @@
+# Code de Conduite Contributeurs
+
+En tant que contributeurs et responsables de ce projet, et dans l'intérêt
+de favoriser une communauté ouverte et accueillante, nous nous engageons à
+respecter toutes les personnes qui contribuent en rapportant des erreurs,
+postant des demandes de fonctionnalités nouvelles, mettant à jour la
+documentation, soumettant des _pull requests_ ou des _patches_, ainsi que
+toute autres activité.
+
+Nous sommes déterminés à faire de la participation dans ce projet une
+expérience exempt de harcèlement pour tout le monde, quel que soit le niveau
+d'expérience, le sexe, l'identité ou l'expression de genre, l'orientation
+sexuelle, le handicap, l'apparence personnelle, la taille physique, la race,
+l'origine ethnique, l'âge, la religion, ou la nationalité.
+
+Des exemples de comportement inacceptable par les participants comprennent:
+
+* L'utilisation de language ou d'imagerie sexualisée
+* Les attaques personnelles
+* Les _trolls_ ou commentaires insultants ou désobligeants
+* Le harcèlement public ou privé
+* La publication d'autres informations privées, telles que les adresses
+  physiques ou électroniques, sans permission explicite
+* Toute autre conduite contraire à l'éthique ou non professionnelle
+
+Les mainteneurs du projet ont le droit et la responsabilité de supprimer,
+modifier ou rejeter les commentaires, _commits_, code, modifications du wiki,
+questions et autres contributions qui ne respectent pas ce Code de Conduite,
+ou de bannir temporairement ou définitivement tout contributeur à la suite
+d'autres comportements qui qu'ils jugent inapproprié, menaçant, injurieux,
```

Comment:
Mmm... not sure, re-reading it now, I feel it would read better in the plural as it refers to `comportements`. The `qui` is definitely superfluous though.